### PR TITLE
Updated descriptions for bookingId, supplierConfirmationNumber, hotel…

### PR DESCRIPTION
**Pull request by rohit77p**
Updated descriptions for bookingId, supplierConfirmationNumber, hotelConfirmationNumber, supplierCancellationNumber, and hotelCancellationNumber fields.

### CODE REVIEW

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 1]
```diff
@@ -6,7 +6,10 @@
 + include contentPrefs
 
 ## hotelBookingCancellationRequest
-+ bookingId: `BT244353` (string, required) - Unique ID of the booking that you want to cancel. You must obtain the `bookingId` for a booking from the `Book` API's response.
++ bookingId: `BT244353` (string, required) - Unique ID of the booking that you want to cancel. You must obtain the `bookingId` for a booking from the `Book` API's response.<br>
+<b>Notes:</b>
+  + The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs).
+  + The `bookingId` is generated by Connexions API and is not shared with the hotel, which means that the `bookinId` cannot be used as a confirmation number for hotel reservation.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Please update bookingId second instance typo and the verbiage update as follows:
The Connexions API generates the `bookingId`. This ID is not shared with the hotel, which means that the `bookingId` cannot be used as a confirmation number for a hotel reservation.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 2]
```diff
@@ -53,17 +62,30 @@
...
 + roomId: `828a7e10-98de-4b4f-a533-572132ecacf8` - Room ID for which the cancellation was made.
 + rateId: `aa0123aa-12z3-1234-8f12-1236374qwz12` - Rate reference ID associated with the room. Each `rateId` represents a rate for a valid combination of a room type and occupancy.
-+ supplierCancellationNumber: `78558816` - Cancellation number received from the supplier. If the supplier did not return any cancellation number, this field is set to `null`.
-+ hotelCancellationNumber: `8742261` - Cancellation number received from the hotel. If the hotel did not return any cancellation number, this field is set to `null`.
++ include supplierCancellationNumber
++ include hotelCancellationNumber
 + errors (array[errors]) - Contains the list of errors that occurred during the cancellation of the room.
 
+## supplierCancellationNumber
++ supplierCancellationNumber: `78558816` - Supplier cancellation number for the booking. For every successful booking cancellation, the suppliers cancellation number is returned, either as the `/cancellationDetails/supplierCancellationNumber` or as the `/cancellationDetails/roomCancellations[]/supplierCancellationNumber` field.<br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: For every successful cancellation, the supplier's cancellation number is returned either in the `/cancellationDetails/supplierCancellationNumber` field or the `/cancellationDetails/roomCancellations[]/supplierCancellationNumber` field.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 3]
```diff
@@ -53,17 +62,30 @@
...
+<b>Notes:</b>
+  + The `supplierCancellationNumber` is required when you contact Connexions Loyalty Customer Service.
+  + The `supplierCancellationNumber` should be shared with the guests to be used as cancellation number for hotel reservation.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: The `supplierCancellationNumber` must be used as the cancellation number for the hotel reservation.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 4]
```diff
@@ -72,14 +94,20 @@
...
+  + The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs).
+  + The `bookingId` is generated by Connexions API and is not shared with the hotel, which means that the `bookinId` cannot be used as a confirmation number for hotel reservation.
 
 ## bookingStatusResponse
 + include bookingIdResStatus
 
 
 ## bookingIdResStatus
-+ bookingId: `BW234223` - Unique ID for the booking. You need to use the `bookingId` as an identifier for the booking in other API calls at a later point in time. For example, you will need the `bookingId` when you want to request for a booking cancellation.
++ bookingId: `BW234223` - Unique ID for the booking. You need to use the `bookingId` as an identifier for the booking in other API calls at a later point in time. For example, you will need the `bookingId` when you want to request for a booking cancellation.<br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: update all instances of "need to" to "must"

For example, you must use the `bookingId` when you request for a

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 5]
```diff
@@ -72,14 +94,20 @@
...
+<b>Notes:</b>
+  + The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs).
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: same updates as earlier instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 6]
```diff
@@ -72,14 +94,20 @@
...
+  + The `bookingId` is generated by Connexions API and is not shared with the hotel, which means that the `bookinId` cannot be used as a confirmation number for hotel reservation.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: same updates as earlier instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 7]
```diff
@@ -186,10 +220,16 @@
 + include bookingIdReq
 
 ## bookingIdRes
-+ bookingId: `BW234223` - Unique ID for the booking. You need to use the `bookingId` as an identifier for the booking in other API calls at a later point in time. For example, you will need the `bookingId` when you check the booking status.<br><br><b>Note:</b> The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs). The `bookingId` is not shared with the hotel.
++ bookingId: `BW234223` - Unique ID for the booking. You need to use the `bookingId` as an identifier for the booking in other API calls at a later point in time. For example, you will need the `bookingId` when you check the booking status.<br>
+<b>Notes:</b>
+  + The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs).
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: same edits as first instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 8]
```diff
@@ -186,10 +220,16 @@
...
+  + The `bookingId` is generated by Connexions API and is not shared with the hotel, which means that the `bookinId` cannot be used as a confirmation number for hotel reservation.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: same edits as first instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 9]
```diff
@@ -186,10 +220,16 @@
...
 
 ## bookingIdReq
-+ bookingId: `BW234223` - Unique ID of the booking for which you want to check the status. You must obtain the `bookingId` for a booking from the `Book` API's response.
++ bookingId: `BW234223` - Unique ID of the booking for which you want to check the status. You must obtain the `bookingId` for a booking from the `Book` API's response.<br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: same edits as first instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 10]
```diff
@@ -186,10 +220,16 @@
...
+<b>Notes:</b>
+  + The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs).
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: same edits as first instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 11]
```diff
@@ -186,10 +220,16 @@
...
+  + The `bookingId` is generated by Connexions API and is not shared with the hotel, which means that the `bookinId` cannot be used as a confirmation number for hotel reservation.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: same edits as first instance

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 3 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 12]
```diff
@@ -53,17 +62,30 @@
...
 + errors (array[errors]) - Contains the list of errors that occurred during the cancellation of the room.
 
+## supplierCancellationNumber
++ supplierCancellationNumber: `78558816` - Supplier cancellation number for the booking. For every successful booking cancellation, the suppliers cancellation number is returned, either as the `/cancellationDetails/supplierCancellationNumber` or as the `/cancellationDetails/roomCancellations[]/supplierCancellationNumber` field.<br>
+<b>Notes:</b>
+  + The `supplierCancellationNumber` is required when you contact Connexions Loyalty Customer Service.
+  + The `supplierCancellationNumber` should be shared with the guests to be used as cancellation number for hotel reservation.
+
+## hotelCancellationNumber
++ hotelCancellationNumber: `8742261` - Hotel cancellation number for the booking. This field will return `null` if the hotel has not provided a cancellation number. If available, the `hotelCancellationNumber` is returned either as the `/cancellationDetails/hotelCancellationNumber` or as the `/cancellationDetails/roomCancellations[]/hotelCancellationNumber` field.<br>
```
[the remark this answers is not in the document]
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 13]
```diff
@@ -53,17 +62,30 @@
...
+<b>Notes:</b>
+  + If available, the `hotelCancellationNumber` is required when you contact Connexions Loyalty Customer Service.
```
[the remark this answers is not in the document]
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 14]
```diff
@@ -53,17 +62,30 @@
...
+  + If available, the `hotelCancellationNumber` along with the `supplierCancellationNumber` should be shared with the guests to be used as cancellation number for hotel reservation.
```
[the remark this answers is not in the document]
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 15]
```diff
@@ -72,14 +94,20 @@
 + fees (array[feescancellation]) - Contains the list of all the applicable fees for the cancellation. This amount is not included in the `baseAmount`. However, this amount is already included in the `totalAmount`.
 
 ## bookingStatusRequest
-+ bookingId: `BW234223` (string, required)- Unique ID of the booking for which you want to check the status. You must obtain the `bookingId` from `Book` API's response.
++ bookingId: `BW234223` (string, required)- Unique ID of the booking for which you want to check the status. You must obtain the `bookingId` from `Book` API's response.<br>
```
[the remark this answers is not in the document]
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 16]
```diff
@@ -72,14 +94,20 @@
...
+<b>Notes:</b>
+  + The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs).
+  + The `bookingId` is generated by Connexions API and is not shared with the hotel, which means that the `bookinId` cannot be used as a confirmation number for hotel reservation.
```
[the remark this answers is not in the document]
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 17]
```diff
@@ -19,15 +22,21 @@
...
 ## hotelBookingCancellationStatusResponse
 + cancellationProgress (enum) - Indicates the status of the cancellation request.
   + InProgress
   + Completed
   + CompletedWithFailures
-+ bookingId: `BT244353` - Booking ID for which the cancellation request was made.
++ bookingId: `BT244353` - Booking ID for which the cancellation request was made.<br>
+<b>Notes:</b>
+  + The `bookingId` is used only for checking the booking status (by using the `Get Booking Status` API) and canceling the booking (by using the `Initiate Cancellation` and `Get Cancellation Status` APIs).
+  + The `bookingId` is generated by Connexions API and is not shared with the hotel, which means that the `bookinId` cannot be used as a confirmation number for hotel reservation.
```
[the remark this answers is not in the document]
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 4 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated all relevant instances.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 5 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 6 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 7 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 8 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 10 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib (hunk 11 above)
Comment (rohit77p) [reply apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: Updated as suggested.

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 18]
```diff
@@ -6,7 +6,10 @@
 + include contentPrefs
 
 ## hotelBookingCancellationRequest
-+ bookingId: `BT244353` (string, required) - Unique ID of the booking that you want to cancel. You must obtain the `bookingId` for a booking from the `Book` API's response.
++ bookingId: `BT244353` (string, required) - Unique ID of the booking that you want to cancel. You must obtain the `bookingId` for a booking from the `Book` API's response.<br>
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: bookingId: `BT244353` (string, required) - Unique ID of the booking that you want to cancel. You must obtain the `bookingId` from the `Book` API's response.<br>

File: apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib [hunk 19]
```diff
@@ -82,13 +82,14 @@
 <b>Notes:</b>
   + The `supplierCancellationNumber` is required when you contact Connexions Loyalty Customer Service.
   + The `supplierCancellationNumber` must be used as the cancellation number for the hotel reservation.
+  + If the supplier cancellation number is returned both in the `/cancellationDetails/supplierCancellationNumber` field and in the `/cancellationDetails/roomCancellations[]/supplierCancellationNumber` field, the number returned in the `/cancellationDetails/supplierCancellationNumber` field must be used as the cancellation number for the hotel reservation.
```
Comment (mirairani) [inline apis/booking-engine/hotel/v1.0/_dataStructure/data-structures.apib]: remove the word both as we are already using and

**Pull request merged**